### PR TITLE
doc(readme): add explanation for retrieving results

### PR DESCRIPTION
**Pull request by Marr11317**
see #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +50,9 @@ use it to create an another task for that instance.
 Note that the task name should always be unique in an instance, because Tinybench stores the tasks based
 on their names in a `Map`.
 
+Also note that `tinybench` does not log any result by default. You can extract the relevant stats
+from `bench.results` after running the benchmark, and process them however you want.
```
Comment (Aslemammad) [inline README.md:71]: `Task.results` and not bench.results.

File: README.md [hunk 2]
```diff
@@ -39,6 +39,8 @@ bench
   });
 
 await bench.run();
+
+// view, print, process bench.results
```
Comment (Aslemammad) [inline README.md]: let's remove this comment here, the note is enough.

Comment (ecstrema) [reply README.md]: I believe the best would be to add an actual example of how to treat the results. 

That way the example would be complete.

Comment (Aslemammad) [reply README.md]: That's also good!

File: README.md (hunk 1 above)
Comment (ecstrema) [reply README.md:71]: Well Task.result, but Bench.result**s** if I read well the documentation below.

File: README.md (hunk 2 above)
Comment (Aslemammad) [reply README.md]: try getting the task (with bench.getTask) and log its mean or hz! that example here is good

File: README.md (hunk 1 above)
Comment (Aslemammad) [reply README.md:71]: Oh i thought you just wanted to process one task result! it's okay, you can go with bench.results if you want to process all results.

File: README.md (hunk 2 above)
Comment (Aslemammad) [reply README.md]: btw, it'd be good if we can convert the require call in the example to an import call! because it's incorrect! Thanks.

Comment (ecstrema) [reply README.md]: done

**Pull request merged**